kolibri/core/discovery/upgrade.py

```diff
@@ -0,0 +1,35 @@
+"""
+A file to contain specific logic to handle version upgrades in Kolibri.
+"""
+import logging
+
+from django.db.utils import OperationalError
+
+from kolibri.core.discovery.models import NetworkLocation
+from kolibri.core.upgrade import version_upgrade
+from kolibri.deployment.default.sqlite_db_names import NETWORK_LOCATION
+
+logger = logging.getLogger(__name__)
+
+
+@version_upgrade(old_version="<0.15.0")
+def move_network_location_entries():
```

Comment:
As we're doing the upgrade (for the static locations only, I suppose) we could remove this table from the previous database as Django won't do it.
In new installations of kolibri the table won't exist anyway.

Reply:
Can give that a go, yes!